### ISSUE TITLE
Info about the pretrained models

### ISSUE THREAD

**Issue by yossigandelsman** [pengzhiliang/MAE-pytorch#18]
Can you please give more info about the pretrained models?
- How many warmup epochs did you use?
- How many GPUs, what kind, and what was the batch size and the learning rate?

Thank you very much!

**Comment by pengzhiliang** [comment]
Ok, thanks.

> How many warmup epochs did you use?

We used 40 epoch for warmup.

> How many GPUs, what kind, and what was the batch size and the learning rate?

We only used 8 v100s. And the batch size is 512 * 8, the lr is 512 * 8 *1.5e-4 / 256 (lr scaling rule according to the paper)
